metagen-lib-basics/src/main/java/io/virtdata/basicsmappers/from_long/to_string/DirectoryLines.java

```diff
@@ -66,7 +66,7 @@ public synchronized String apply(long value) {
             throw new RuntimeException(e);
         }
         logger.debug("File reader: " + fileList.toString() + " in path: " + Paths.get(basepath).getFileName());
-
+        fileList.paths.sort(Path::compareTo);
```

Comment:
This is not in scope with the comment for the PR. Not a big deal here, but I'll tend to merge PRs that are narrowly themed more quickly.

Reply:
It is already explained in the PR description why i made this change, otherwise i would not have been able to build without having the tests fail on me.
I kept the change on a separate commit.